File: src/Health.Core/Entities/Food.cs
using System.ComponentModel.DataAnnotations;

namespace Health.Core.Entities
{
    public class Food
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        public int Calories { get; set; }
        public int Protein { get; set; }
        public int Fat { get; set; }
        public int Carbs { get; set; }
        public int? Sugar { get; set; }
        public int ServingSize { get; set; }
        public string ServingName { get; set; }
        public int? Fiber { get; set; }
        public int? Sodium { get; set; }
        public int? Potassium { get; set; }
    }
}

File: src/Health.Web/app/classes/food.ts
class Food implements IFood
{
    public Id: number;
    public Name: string;
    public Calories: number;
    public Protein: number;
    public Fat: number;
    public Carbs: number;
    public Sugar: number;
    public ServingSize: number;
    public ServingName: string;
    public Fiber: number;
    public Sodium: number;
    public Potassium: number;

    constructor(food: IFood)
    {
        this.Id = food.Id;
        this.Name = food.Name;
        this.Calories = food.Calories
        this.Protein = food.Protein;
        this.Fat = food.Fat;
        this.Carbs = food.Carbs;
        this.Sugar = food.Sugar;
        this.ServingSize = food.ServingSize;
        this.ServingName = food.ServingName;
        this.Fiber = food.Fiber;
        this.Sodium = food.Sodium;
        this.Potassium = food.Potassium;
    }
}